packages/admin/blocks-admin/src/blocks/factories/createListBlock.tsx

```diff
@@ -113,22 +133,12 @@ export function createListBlock<T extends BlockInterface, AdditionalItemFields e
         displayName,
 
         defaultValues: () => ({
-            blocks: createDefaultListEntry
-                ? [
-                      {
-                          key: uuid(),
-                          visible: true,
-                          props: block.defaultValues(),
-                          selected: false,
-                          slideIn: false,
-                          // Type cast to suppress "'AdditionalItemFields' could be instantiated with a different subtype of constraint 'Record<string, unknown>'" error
-                          ...(Object.entries(additionalItemFields ?? {}).reduce(
-                              (fields, [field, { defaultValue }]) => ({ ...fields, [field]: defaultValue }),
-                              {},
-                          ) as AdditionalItemFields),
-                      },
-                  ]
-                : [],
+            blocks:
+                minVisibleBlocks !== undefined
+                    ? Array.from({ length: minVisibleBlocks }, () => getDefaultListEntry())
```

Comment:
nit: I believe the wrapper function is unnecessary

```suggestion
                    ? Array.from({ length: minVisibleBlocks }, getDefaultListEntry)
```